scripts/sync-dashbaords.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DASHBOARDS="apm-dependencies.json apm-home.json apm-service-dependencies-downstream.json apm-service-dependencies-upstream.json apm-service-overview.json promscale.json"
```

Comment:
why do we want an explicit list instead of downloading all of them?

Reply:
No reason at all. I rewrote it to download all dashboards.